bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsHash.scala

```diff
@@ -59,13 +59,13 @@ object AgsHash {
     // Base Position
     Option(ctx.getTargets).foreach { t =>
       val time = Some(new java.lang.Long(when)).asGeminiOpt
-      val base = t.getBase
+      val asterism = t.getAsterism
 
       def toData(coord: GemOption[java.lang.Double]): Int =
         coord.asScalaOpt.map(_.doubleValue).##
 
-      val ra  = toData(base.getRaDegrees(time))
-      val dec = toData(base.getDecDegrees(time))
+      val ra  = toData(asterism.getRaDegrees(time))
```

Comment:
I haven't done it yet, and I may not because we don't have an asterism with more than two targets yet.  The GHOST dual target base position is calculated by interpolating the two targets with `Coordinates.interpolate`.

My thought though was that I would pick a random target as a base position (0,0) and then calculate the angular separation between it and the other targets as an `Offset` with `Coordinates.difference(c0, c1).offset`.  In other words, the curvature comes into account when calculating the offset and when converting back to RA/Dec but the smallest circle calculation would work with the projection of the points on a plane.  It's likely I'm confused though, which is another good reason to wait until we have a 3+ target asterism.